addons/mail/views/mail_message_views.xml

```diff
@@ -99,18 +99,20 @@
                     <field name="name"/>
                     <templates>
                         <t t-name="kanban-box">
-                            <div class="oe_kanban_global_click">
+                            <div>
                                 <div class="oe_kanban_vignette">
                                     <div class="oe_attachment">
-                                        <div t-if="record.file_type_icon.value != 'webimage'">
-                                            <img t-att-src="'/mail/static/src/img/mimetypes/' + record.file_type_icon.value + '.png'"></img>
-                                        </div>
-                                        <div t-if="record.file_type_icon.value == 'webimage' and !record.url.value">
-                                            <img t-att-src="kanban_image('ir.attachment', 'datas', record.id.value)"/>
-                                        </div>
-                                        <div t-if="record.file_type_icon.value and record.url.value">
-                                            <img t-att-src="record.url.value"/>
-                                        </div>
+                                        <a t-att-href="kanban_image_saveas('ir.attachment', 'datas', 'datas_fname', record.id.value)">
```

Comment:
@tde-banana-odoo, 
As per our requirement for digital products, when going to the digital product form to
<code> ir.attachment </code> Kanban view, then after click on the Kanban card, so that attachment of digital product should be downloaded.